### PR TITLE
[DEV-1039] Pull retag push fix

### PR DESCRIPTION
**Pull request by g0rgamesh**
- changed pull location to new one in quay.io
- changed tag naming to correct one

### CODE REVIEW

**Review by priteau — CHANGES_REQUESTED** [review]
Please fix the typo in the commit message. Maybe add a small description.
Add that it closes #126

**Review by priteau — CHANGES_REQUESTED** [review]
I was thinking of adding this new information to the commit rather than the PR.

**Review by priteau — APPROVED** [review]

File: etc/kayobe/ansible/pull-retag-push.yml [hunk 1]
```diff
@@ -80,24 +80,24 @@
     - name: Pull container images (may take a long time)
       become: true
       vars:
-        remote_image: "{% if docker_pull_registry != '' %}{{ docker_pull_registry }}/{% endif %}{{ docker_pull_namespace }}/{{ item }}:{{ docker_pull_tag }}"
+        remote_image: "{% if docker_pull_registry != '' %}{{ docker_pull_registry }}/{% endif %}{{ docker_pull_namespace }}/{{ item }}:{{ docker_pull_tag }}-{{ kolla_base_distro }}-{{ ansible_facts.distribution_release }}"
```
Comment (markgoddard) [inline etc/kayobe/ansible/pull-retag-push.yml:83]: I don't think ansible_facts.distribution_release is always correct here. For example, on CentOS stream it is `stream8`.

Comment (g0rgamesh) [reply etc/kayobe/ansible/pull-retag-push.yml:83]: Would ansible_distribution_release be better?

Comment (markgoddard) [reply etc/kayobe/ansible/pull-retag-push.yml:83]: That's just a different way of accessing the same thing. Here's how kolla-ansible does it:

https://opendev.org/openstack/kolla-ansible/src/commit/a396284a83e7624a951190b36fbad79812f1937d/ansible/group_vars/all.yml#L1196

**Pull request merged**